src/compat/eme/custom_media_keys/webkit_media_keys.ts

```diff
@@ -123,9 +123,12 @@ class WebkitMediaKeySession
           return reject("Unavailable WebKit key session.");
         }
         try {
+          const licenseTypedArray = license instanceof ArrayBuffer
```

Comment:
We might want to re-define `update` as accepting an ArrayBuffer if we're going down this path.

You also named it `licenseTypedArray` which seems to me a too precize name for not being not enough precize: if we name it like that, not only is it a TypedArray, it is a Uint8Array specifically, not any other one.

Also on the same point, what if the license given was a Uint16Array, or a Float64Array? Those are all valid BufferSource but still not a Uint8Array.